examples/ExactlyOnceBasic/Program.cs

```diff
@@ -0,0 +1,707 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using Confluent.Kafka;
+using Confluent.Kafka.Admin;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+/// <summary>
+///     This example demonstrates a basic Exactly-once semantics producer moving messages from
+///     an input topic to an output topic. To run the example follow these steps: 
+///
+///     1. create input topic with: createtopic <input_topic> <num_partitions>
+///     2. create output topic with: createtopic <output_topic> <num_partitions>
+///     3. generate messages and send them to the input topic: generate <input_topic> <msgs_per_second>
+///     4. process messages and copy them to the output topic: process <input_topic> <output_topic> <commit_period_seconds>
+/// </summary>
+namespace Confluent.Kafka.Examples.ExactlyOnceBasic
+{
+
```

Comment:
have a look at other examples for coding style: how many new lines to put where ...